src/app_defs.py

```diff
@@ -6,10 +6,10 @@
 import logging
 from typing import List
 
-APP_NAME_SHORT = 'znode-tool'
-APP_NAME_LONG = 'Znode Tool'
-APP_DATA_DIR_NAME = '.znode-tool'
-PROJECT_URL = 'https://github.com/zcoinofficial/znode-tool-evo'
+APP_NAME_SHORT = 'masternode-tool'
+APP_NAME_LONG = 'Masternode Tool'
+APP_DATA_DIR_NAME = '.masternode-tool'
```

Comment:
I'm not sure about changing default config dir here. Old version data will remain in .znode-tool  and some guidance will be required to rename it to .masternode-tool.
Maybe I shouldn't touch data dir at all ?

Reply:
Adding to the confusion, znode-tool (legacy) uses `.znode-tool`, znode-tool-evo uses `.znode-tool-v5`

Reply:
v5 is added by the flatpak package handlers.

Reply:
@justanwar, do you think we should leave the config as .znode-tool for convinience?

Reply:
I think we can bite the bullet and go with `.firo-masternode-tool` otherwise we will never move on from znode. I will make a note of this in the release notes/guides.